Gems/Terrain/Code/Source/TerrainRenderer/TerrainFeatureProcessor.cpp

```diff
@@ -1438,6 +1630,27 @@ namespace Terrain
             }
         }
     }
+
+    void TerrainFeatureProcessor::CacheForwardPass()
+    {
+        auto rasterPassFilter = AZ::RPI::PassFilter::CreateWithPassClass<AZ::RPI::RasterPass>();
+        rasterPassFilter.SetOwenrScene(GetParentScene());
```

Comment:
"SetOwenrScene"?  Not a typo?

Reply:
I pointed this out to Vicky and she put in a fix. I'll update when I merge latest from dev.